src/screen_credits.c

```diff
@@ -26,6 +26,63 @@ static const char *creditstxt[] = {
     "luksamuk",
     "\r",
 
+    /* SFX */
+    "Sound Effects Ripping",
+    "luksamuk",
+    "Mr. Lange",
+    "\r",
+
+
+    /* Graphics credits */
+    /* "Character Sprites Ripping", */
```

Comment:
Remove these comments. All other objects are by Paraemon, so no need for character sprites stuff here